diff --git a/node_modules/@codspeed/core/dist/index.cjs.js b/node_modules/@codspeed/core/dist/index.cjs.js
index 1c40cda..4a5d588 100644
--- a/node_modules/@codspeed/core/dist/index.cjs.js
+++ b/node_modules/@codspeed/core/dist/index.cjs.js
@@ -26,7 +26,10 @@ const getV8Flags = () => {
     "--no-opt",
     "--predictable",
     "--predictable-gc-schedule",
-    "--interpreted-frames-native-stack"
+    "--interpreted-frames-native-stack",
+    // "--jitless",
+    '--no-concurrent-sweeping',
+    '--max-old-space-size=4096',
   ];
   if (nodeVersionMajor < 18) {
     flags.push("--no-randomize-hashes");