diff --git a/node_modules/@codspeed/vitest-plugin/dist/runner.mjs b/node_modules/@codspeed/vitest-plugin/dist/runner.mjs
index f4c6c95..ee26264 100644
--- a/node_modules/@codspeed/vitest-plugin/dist/runner.mjs
+++ b/node_modules/@codspeed/vitest-plugin/dist/runner.mjs
@@ -32,6 +32,7 @@ async function runBench(benchmark, currentSuiteName) {
       throw e;
     }
   }
+  global.gc();
   await callSuiteHook(benchmark.suite, benchmark, "afterEach");
   await callSuiteHook(benchmark.suite, benchmark, "beforeEach");
   await mongoMeasurement.start(uri);
