diff --git a/node_modules/@codspeed/core/dist/index.cjs.js b/node_modules/@codspeed/core/dist/index.cjs.js
index 5087584..64fa54a 100644
--- a/node_modules/@codspeed/core/dist/index.cjs.js
+++ b/node_modules/@codspeed/core/dist/index.cjs.js
@@ -25,8 +25,12 @@ const getV8Flags = () => {
     "--random-seed=1",
     "--no-opt",
     "--predictable",
+    "--expose-gc"
     "--predictable-gc-schedule",
-    "--interpreted-frames-native-stack"
+    "--interpreted-frames-native-stack",
+    // "--jitless",
+    '--no-concurrent-sweeping',
+    '--max-old-space-size=4096',
   ];
   if (nodeVersionMajor < 18) {
     flags.push("--no-randomize-hashes");
@@ -18861,6 +18865,9 @@ const optimizeFunction = async (fn) => {
   }
   await fn();
   await fn();
+  await fn();
+  await fn();
+  await fn();
   eval("%OptimizeFunctionOnNextCall(fn)");
   await fn();
 };
@@ -18871,6 +18878,9 @@ const optimizeFunctionSync = (fn) => {
   }
   fn();
   fn();
+  fn();
+  fn();
+  fn();
   eval("%OptimizeFunctionOnNextCall(fn)");
   fn();
 };
